cli/azd/cmd/init.go

```diff
@@ -252,3 +244,22 @@ func (i *initAction) Run(ctx context.Context) (*actions.ActionResult, error) {
 		},
 	}, nil
 }
+
+func getCmdInitHelpDescription(c *cobra.Command) string {
+	return generateCmdHelpDescription(i18nGetText(i18nCmdInitHelp), getCmdHelpDescriptionNoteForInit(c))
+}
+
+func getCmdInitHelpFooter(*cobra.Command) string {
+	return generateCmdHelpSamplesBlock(map[string]string{
+		i18nGetText(i18nCmdInitHelpSample): fmt.Sprintf("%s %s",
+			output.WithHighLightFormat("azd init --template"),
+			output.WithWarningFormat("[GitHub repo URL]"),
+		),
+		i18nGetText(i18nCmdInitHelpSampleBranch): fmt.Sprintf("%s %s %s %s",
+			output.WithHighLightFormat("azd init --template"),
+			output.WithWarningFormat("[GitHub repo URL]"),
+			output.WithHighLightFormat("--branch"),
+			output.WithWarningFormat("[Branch name]"),
+		),
+	})
+}
```

Comment:
It is a bit off putting that we need to manage things with special formatting as a special case vs standard text.  Is there a way we could use/introduce some templating language within the yaml document that would allow us to output the next with our built in formatting?